python/ray/data/preprocessors/vectorizer.py

```diff
@@ -9,15 +9,84 @@
 
 
 class HashingVectorizer(Preprocessor):
-    """Tokenize and hash string columns into token hash columns.
+    """Count the frequency of tokens using the
+    `hashing trick <https://en.wikipedia.org/wiki/Feature_hashing>`_.
 
-    The created columns will have names in the format ``hash_{column_name}_{hash}``.
+    This preprocessors creates ``num_features`` columns named like
+    ``hash_{column_name}_{index}``. If ``num_features`` is large enough relative to
+    the size of your vocabulary, then each column approximately corresponds to the
+    frequency of a unique token.
+
+    This preprocessor is memory efficient and quick to pickle. However, given a transformed
+    column, you can't know which tokens correspond to it. This might make it hard
+    to determine which tokens are important to your model.
+
+    .. note::
+
+        A document-term matrix is a table that describes the frequency of tokens
```

Comment:
Should document-term matrix be mentioned in the section immediately above this note?

Reply:
Eh...maybe. It's jargon you don't necessarily need to know to understand the preprocessor. Also, wasn't sure how to introduce it in a non-awkward way.